Fastfile

```diff
@@ -164,14 +157,24 @@ def _get_team_id()
   team_id
 end
 
-def _parse_provision_file()
-  sh 'RUBYSCRIPT="$(echo \'def target(targetName, &doBlock)
-    profileName = doBlock.call
-    print targetName + "_" + "PROVISIONING_PROFILE = " + profileName + "\n"
+def _create_xcconfig(filename)
```

Comment:
It might be nice to delete any existing xcconfig file with the same filename because it just appends to the end unless you reset git.